notebooks/cugraph_benchmarks/nx_cugraph_benchmark.ipynb

```diff
@@ -4,13 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Benchmarking Performance of NetworkX with Rapids GPU-based nx_cugraph backend vs on cpu\n",
+    "# Benchmarking Performance of NetworkX with and without the RAPIDS GPU-based nx_cugraph backend\n",
     "# Skip notebook test\n",
-    "This notebook demonstrates compares the performance of nx_cugraph as a dispatcher for NetworkX algorithms. \n",
+    "This notebook collects the run times with and without the nx_cugraph backend and graph caching enabled for three popular NetworkX algorithms: Betweenness Centrality, Breadth First Search, and Louvain Community Detection.\n",
     "\n",
-    "We do this by executing Betweenness Centrality, Breadth First Search and Louvain Community Detection, collecting run times with and without nx_cugraph backend and graph caching enabled. nx_cugraph is a registered NetworkX backend. Using it is a zero code change solution.\n",
-    "\n",
-    "In the notebook switching to the nx-cugraph backend is done via variables set using the [NetworkX config package](https://networkx.org/documentation/stable/reference/backends.html#networkx.utils.configs.NetworkXConfig) **which requires networkX 3.3 or later !!**\n",
+    "In this notebook, enabling the nx-cugraph backend will be done via using the [NetworkX config API](https://networkx.org/documentation/stable/reference/backends.html#networkx.utils.configs.NetworkXConfig) (which requires NetworkX 3.3 or later)\n",
```

Comment:
I think "via" and "using" are redundant.

Reply:
Thanks! Fixed.